### PR TITLE
Encoder manager

### PR DESCRIPTION
**Pull request by AitorTeran**
The things related to the error manager are probably already merged. I had to make the changes to make it work for me.

### CODE REVIEW

File: Software/Development/SemesterProject/System/scheduler.c [hunk 1]
```diff
@@ -209,6 +209,9 @@ void handleSystemClock(void)
     if (sysClock >= INT_MAX)    sysClock = 0;
 
     UARTIntPrint("TS ", (int)sysClock);
+    //UARTIntPrint("RPM ", (int)readRotorRPM());
```
Comment (murgui) [inline Software/Development/SemesterProject/System/scheduler.c]: Remove comments.

File: Software/Development/SemesterProject/App/Include/analogAcquisitionManager.h [hunk 2]
```diff
@@ -13,7 +13,10 @@
 //
 #include "DSP28x_Project.h"     // Device Headerfile and Examples Include File
 #include "analogSignal.h"
-//#include "controlTask.h"
+#include "digitalOutput.h"
```
Comment (murgui) [inline Software/Development/SemesterProject/App/Include/analogAcquisitionManager.h]: line 16 and 17 not necessary

**Review by murgui — APPROVED** [review]
Nice looking,<hopefully everything will work¿

**Review by NicolaiFransen — APPROVED** [review]
¡Envia-ho!

File: Software/Development/SemesterProject/System/scheduler.c (hunk 1 above)
Comment (AitorTeran) [reply Software/Development/SemesterProject/System/scheduler.c]: Done, also added a card in trello regarding this.
https://trello.com/c/oZngu3oG/146-consider-creating-a-function-for-printing-stuff-through-uart

File: Software/Development/SemesterProject/App/Include/analogAcquisitionManager.h (hunk 2 above)
Comment (AitorTeran) [reply Software/Development/SemesterProject/App/Include/analogAcquisitionManager.h]: Done

**Pull request merged**